app/helpers/patients_helper.rb

```diff
@@ -14,6 +14,10 @@ def race_ethnicity_options
      'Native American', 'Mixed Race/Ethnicity', 'Other']
   end
 
+  def language
+    [nil, 'English', 'Spanish', 'French', 'Korean']
```

Comment:
I think English as the best default is a safe-ish assumption -- I don't know of any funds on our prospects list that. If it's possible it may be worth writing this so we can easily inject a default language into it at some point. (see #1092 for a direction we could feasibly take this.)

I defer to @alisonnjones on the analytics question -- she should be at the hacknight tomorrow so we can ask her point blank.